packages/forma-36-react-components/src/components/IconButton/IconButton.css

```diff
@@ -1,13 +1,14 @@
 @import 'settings/colors';
 @import 'settings/typography';
 @import 'settings/transitions';
+@import 'settings/dimensions';
 
 .IconButton {
   composes: focus-outline--default from './../../styles/settings/a11y.css';
   display: inline-block;
   cursor: pointer;
   border: 0;
-  padding: 0;
+  padding: var(--spacing-s);
```

Comment:
Thanks again for the PR @joneskj55 
I just did some quick visual review, I believe we need to fix the following components before we get this in.

I am not sure if we should treat this as a fix, It's more like a breaking change, I'm afraid consumers would run into visual issues upon upgrading.

cc:// @gui-santos @burakukula @denkristoffer @m10l 

**EntryCard**
<img width="755" alt="Screenshot 2020-11-23 at 11 09 44" src="https://user-images.githubusercontent.com/6163988/99951837-ff763c00-2d7e-11eb-8722-1721b5aff727.png">

**EditorToolbarButton**
<img width="281" alt="Screenshot 2020-11-23 at 11 11 30" src="https://user-images.githubusercontent.com/6163988/99951840-00a76900-2d7f-11eb-9fb9-db4762749445.png">

**EntityListItem**
<img width="437" alt="Screenshot 2020-11-23 at 11 12 41" src="https://user-images.githubusercontent.com/6163988/99951842-013fff80-2d7f-11eb-87b0-daffa5b8eda4.png">

**ModalHeader**
<img width="557" alt="Screenshot 2020-11-23 at 11 13 38" src="https://user-images.githubusercontent.com/6163988/99951843-01d89600-2d7f-11eb-8439-2a8db418b9ed.png">

**InlineEntryCard**
<img width="601" alt="Screenshot 2020-11-23 at 11 17 58" src="https://user-images.githubusercontent.com/6163988/99951846-01d89600-2d7f-11eb-8797-16ce16990613.png">

**CardHeader**
<img width="290" alt="Screenshot 2020-11-23 at 11 19 17" src="https://user-images.githubusercontent.com/6163988/99951848-02712c80-2d7f-11eb-8324-7842ec1885d1.png">

**AutoComplete**
<img width="289" alt="Screenshot 2020-11-23 at 11 24 26" src="https://user-images.githubusercontent.com/6163988/99951853-03a25980-2d7f-11eb-9b14-1a3aaef826ab.png">

**NotificationItem**
<img width="358" alt="Screenshot 2020-11-23 at 11 25 57" src="https://user-images.githubusercontent.com/6163988/99951854-03a25980-2d7f-11eb-8b94-82eba27ed243.png">

Reply:
@mshaaban0 yeah, I agree. I guess all of the examples above needs to be adjusted if we would like to move forward.
And yeah, +1 for including this change into the next major release.

Reply:
Hmmm yeahh I think it's a breaking change

but @joneskj55 if you really need this change. You can have it by passing a class to Button using the className prop
https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Button/Button.tsx#L37

Reply:
Thanks @gui-santos!